internal/cli/api/api.go

```diff
@@ -19,13 +19,18 @@ import (
 	"fmt"
 	"io"
 	"os"
-	"regexp"
-	"strings"
 
 	"github.com/iancoleman/strcase"
 	"github.com/mongodb/mongodb-atlas-cli/atlascli/internal/api"
 	"github.com/mongodb/mongodb-atlas-cli/atlascli/internal/log"
 	"github.com/spf13/cobra"
+	"github.com/spf13/pflag"
+)
+
+var (
+	ErrFailedToSetUntouchedFlags         = errors.New("failed to set untouched flags")
```

Comment:
what do untouched flags mean in this context?

Reply:
The code is setting which flags are available.
For example:
- `--groupId`
- `--orgId`

If the user provided `--groupId`, but not `--orgId`, then `--orgId` is considered "untouched"